gencode.c

```diff
@@ -10712,3 +10712,11 @@ gen_atmmulti_abbrev(compiler_state_t *cstate, int type)
 	}
 	return b1;
 }
+
+struct block *gen_offset_adjustment(compiler_state_t *cstate, int n) {
+    struct block *b = new_block(cstate, BPF_JMP|BPF_JA);
```

Comment:
I couldn't think of a better way of generating a "noop" block. I tried `BPF_ALU|BPF_ADD|BPF_K` with a value of 0, but the optimizer turned it into an unimplemented operator with an operand of 0xffff.

This is an odd ball in pcap-filter syntax, since it doesn't "check" for anything; it's an "always true" term that controls parsing rather than checking anything. But without returning a block something like `offset+6 and host 192.0.2.68` would never work, since `and` assumes a block on both sides (as best I can tell). Returning NULL or nothing doesn't really work.